src/main/java/com/st/utopia/cancellationservice/model/Ticket.java

```diff
@@ -0,0 +1,103 @@
+package com.st.utopia.cancellationservice.model;
+
+import java.time.LocalDateTime;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+
+/**
+ * An airport model.
+ *
+ * @author Henry Cho
+ * 
+ * Model used for the purpose of the cancellation microservice.
+ * 
+ * There are no set methods for information about the ticket such as flight number, row/seat, or class.
+ * 
+ * Upon cancelling the ticket, the microservice needs to set the following fields as null:
+ * 	-reserver
+ * 	-price
+ * 	-reservation_timeout
+ * 	-booking_id
+ * 
+ * This model does not need to define a composite key since the cancellation microservice will search for the ticket via booking_id which is unique.
+ */ 
+
+@Entity
+@Table(name="tbl_tickets")
+public class Ticket{
+	@Column(name="flight")
+	Integer flight;
```

Comment:
You're going to need a class encapsulating the flight, row, and seat for your JPA DAO anyway; making it now both anticipates that necessity and avoids having *way* too many fields in one class (CodeBeat grades the quality of this file as F simply because of that and having too many parameters in the constructor) until you do.